client/src/parser.js

```diff
@@ -0,0 +1,99 @@
+const { parse } = require('path');
+
+/**
+ * @description Parses an JSON created from the .bpmn xml of the model to an executable .robot file
+ * @returns {string} Code that has to be put in .robot file
+ */
+
+// this should be refactored to a dictionary
+function taskLib(app) {
+  if (app == 'Excel.Files') return 'RPA.Excel.Files';
+  if (app == 'Browser') return 'RPA.Browser';
+}
```

Comment:
I would prefer to load this information from our database. What do you think?

Reply:
I don't even see the benefit of this method... Why not simply `return 'RPA.' + app`

Reply:
Currently `return 'RPA.' + app` would suffice. I will fix it.